packages/vscode-ui5-language-assistant-bas-ext/scripts/copy.js

```diff
@@ -0,0 +1,30 @@
+const { copySync, readdirSync } = require("fs-extra");
+const { join } = require("path");
+
+const getBaseSrc = () =>
+  join(__dirname, "..", "..", "vscode-ui5-language-assistant");
+const getBaseDes = () => join(__dirname, "..");
+
+const srcReadme = join(getBaseSrc(), "README.md");
+const destinationReadme = join(getBaseDes(), "README.md");
+console.log(`Copying from ${srcReadme} to ${destinationReadme}`);
+copySync(srcReadme, destinationReadme);
+console.log("Copying finished successfully");
+
+const vsixFiles = [];
+readdirSync(getBaseSrc()).forEach((item) => {
```

Comment:
If you are going to copy only one file
Why do you scan and **collect** all *.vsix file names?

And in this case should not it be an error if multiple *.vsix files are found?

Reply:
because file name e.g `vscode-ui5-language-assistant-3.3.1.vsix` is not clear und therefore checks base folder  to get one. 

Regarding multiple vsix files, it is usually never the case on CI, but anyway I adapted logic which throws an error in case of multiple or when found not vsix file.